docs/data-sources/item.md

```diff
@@ -42,7 +42,8 @@ data "onepassword_item" "example" {
 - `id` (String) The Terraform resource identifier for this item in the format `vaults/<vault_id>/items/<item_id>`.
 - `password` (String, Sensitive) Password for this item.
 - `port` (String) (Only applies to the database category) The port the database is listening on.
-- `private_key` (String, Sensitive) SSH Private Key for this item.
+- `private_key` (String, Sensitive) SSH Private Key in PKCS#8 for this item.
+- `private_key_openssh` (String, Sensitive) SSH Private key in OpenSSH PEM format.
```

Comment:
Though technically correct, I think calling it just "OpenSSH format" suffices here:
```suggestion
- `private_key_openssh` (String, Sensitive) SSH Private key in OpenSSH format.
```